app/views/admin/infobase/_order_form.html.haml

```diff
@@ -0,0 +1,23 @@
+%table.ui.very.basic.table
+  %thead
+    %tr
+      %th User Name
+      %th User Email
+      %th Delivery Address
+      %th Order Items
+      %th Date of Order
+      %th Price of Order
+  %tbody
+  - orders.includes(:user).each do |order|
+    %tr
+      %td #{order.user.name}
```

Comment:
same here, create decorator for order using draper
then delegate name and email to user.
this looks like a violation of Law of Demeter, google it